src/test/java/edu/cornell/kfs/concur/batch/fixture/ConcurRequestExtractFileFixture.java

```diff
@@ -13,27 +13,54 @@
+             ConcurRequestExtractRequestDetailFileLineFixture.REQUEST_DETAIL_GOOD_LINE_3.createConcurRequestExtractRequestDetailFileLine(),
+             ConcurRequestExtractRequestDetailFileLineFixture.REQUEST_DETAIL_GOOD_LINE_4.createConcurRequestExtractRequestDetailFileLine(),
+             ConcurRequestExtractRequestDetailFileLineFixture.REQUEST_DETAIL_GOOD_LINE_5.createConcurRequestExtractRequestDetailFileLine()}),
+
+    BAD_REQUEST_AMOUNT_FILE("BAD_REQUEST_AMOUNT_FILE", (new Date(Calendar.getInstance().getTimeInMillis())), new Integer(13), new KualiDecimal(9.87),
+        new ConcurRequestExtractRequestDetailFileLine[]
+            {ConcurRequestExtractRequestDetailFileLineFixture.REQUEST_DETAIL_GOOD_LINE_1.createConcurRequestExtractRequestDetailFileLine(),
+             ConcurRequestExtractRequestDetailFileLineFixture.REQUEST_DETAIL_GOOD_LINE_2.createConcurRequestExtractRequestDetailFileLine(),
+             ConcurRequestExtractRequestDetailFileLineFixture.REQUEST_DETAIL_GOOD_LINE_3.createConcurRequestExtractRequestDetailFileLine(),
+             ConcurRequestExtractRequestDetailFileLineFixture.REQUEST_DETAIL_GOOD_LINE_4.createConcurRequestExtractRequestDetailFileLine(),
+             ConcurRequestExtractRequestDetailFileLineFixture.REQUEST_DETAIL_GOOD_LINE_5.createConcurRequestExtractRequestDetailFileLine()}),
     
+    BAD_EMPLOYEE_GROUP_ID_FILE("BAD_EMPLOYEE_GROUP_ID_FILE", (new Date(Calendar.getInstance().getTimeInMillis())), new Integer(13), new KualiDecimal(5253.33),
+        new ConcurRequestExtractRequestDetailFileLine[]
+            {ConcurRequestExtractRequestDetailFileLineFixture.REQUEST_DETAIL_GOOD_LINE_1.createConcurRequestExtractRequestDetailFileLine(),
+             ConcurRequestExtractRequestDetailFileLineFixture.REQUEST_DETAIL_BAD_EMPLOYEE_GROUP_ID_LINE_2.createConcurRequestExtractRequestDetailFileLine(),
+             ConcurRequestExtractRequestDetailFileLineFixture.REQUEST_DETAIL_GOOD_LINE_3.createConcurRequestExtractRequestDetailFileLine(),
+             ConcurRequestExtractRequestDetailFileLineFixture.REQUEST_DETAIL_GOOD_LINE_4.createConcurRequestExtractRequestDetailFileLine(),
+             ConcurRequestExtractRequestDetailFileLineFixture.REQUEST_DETAIL_GOOD_LINE_5.createConcurRequestExtractRequestDetailFileLine()});
+
     public final String fileName;
     public final Date batchDate;
     public final Integer recordCount;
     public final KualiDecimal totalApprovedAmount;
     public final List<ConcurRequestExtractRequestDetailFileLine> requestDetails;
     
-    private ConcurRequestExtractFileFixture (String fileName, Date batchDate, Integer recordCount, KualiDecimal totalApprovedAmount) {
+    private ConcurRequestExtractFileFixture (String fileName, Date batchDate, Integer recordCount,
+                                             KualiDecimal totalApprovedAmount, ConcurRequestExtractRequestDetailFileLine[] detailLines) {
```

Comment:
There are a few things that could help simplify this constructor and the setup of the enum constants:

* You can pass in the date as a String or a Joda DateTime and then convert it to a Date when you're ready to use it in that form. Depending on how you store it in the enum, this can help ensure that your enum constants stay immutable (since Java Date objects are mutable).
* Instead of passing in an Integer object, you can have the constructor accept a primitive int and then have the constructor body wrap it in an Integer.
* Instead of passing in a KualiDecimal, you can have the constructor accept a primitive double and then have the constructor body convert it to a KualiDecimal.
* If the request extract lines are mutable, I would suggest keeping them in their immutable enum form until you're ready to convert them to the needed POJO form. And if you need to store a List of them on this enum, I would strongly recommend using an unmodifiable List, to keep your enum constants truly immutable.